### ISSUE TITLE
misplaced comment after formating

### ISSUE THREAD

**Issue by eMerzh** [jleclanche/tan#1]
hey @jleclanche Many thanks for your fork.
i was trying to use black but got stuck because here we use tabs too.
anyway, 
i sumble upon a formating issue:

Operating system:
Python version: 3.6.5
*Tan* version: latest 
Does also happen on master: yes

The issues is this code : 

```python
def foo():
	# This will eb done
	a = 0
	# Yay
	for i in range(10):
		# Comment the loop
		if a:
			# Break this loop
			break

	# This should
	b = 1
	# Comment inline
	c = 0
	return 0
```

with tabs of courses, 
`tan --use-tabs plop.py`
will be reformated to 

```python
def foo():
	# This will eb done
	a = 0
	# Yay
	for i in range(10):
		# Comment the loop
		if a:
			# Break this loop
			break

			# This should
	b = 1
	# Comment inline
	c = 0
	return 0
```

notice the comment above b=1...

**Comment by jleclanche** [comment]
Yep, that's https://github.com/ambv/black/issues/262 -- I looked into it and found a lead in the code but haven't gotten around fixing it yet. If it gets fixed upstream, I'll backport it :)

**Comment by jleclanche** [comment]
I'm closing the issue here, you can discuss it in the upstream issue I linked.

**Comment by eMerzh** [comment]
Yey , thanks i'll follow that and black's update to tan ;-)

**Comment by jleclanche** [comment]
This should be fixed in 19.1.0.

**Comment by eMerzh** [comment]
awesome, thx for the update
